torch_xla/csrc/ops/ops.cpp

```diff
@@ -783,8 +763,13 @@ torch::lazy::NodePtr Remainder(const XlaValue& input, const XlaValue& divisor) {
   torch::lazy::NodePtr f = Fmod(
       input,
       torch::lazy::MakeNode<Abs>(divisor, std::vector<torch::lazy::Shape>()));
-  return f + divisor * ComparisonOp(at::aten::lt, SignOp(f) * SignOp(divisor),
-                                    ScalarOp(0, input.xla_shape()));
+  return f + divisor * ComparisonOp(
+                           at::aten::lt,
+                           torch::lazy::MakeNode<Sign>(
+                               f, std::vector<torch::lazy::Shape>()) *
+                               torch::lazy::MakeNode<Sign>(
+                                   divisor, std::vector<torch::lazy::Shape>()),
```

Comment:
This kind of call pattern starts to get very messy. I suggest we think about a cleaner solution. Thinking...

Reply:
we need to clean up all of the Node Level lowering. meaning a Node should not use another Node to do the lowering directly. The solution is to wrote a lower function for all these ops and in this case call `BuildSign`.